node/_tools/TODO.md

```diff
@@ -1,2731 +1,2807 @@
-1868. [parallel/test-signal-handler.js](https://github.com/nodejs/node/tree/main/test/parallel/test-signal-handler.js)
-1869. [parallel/test-signal-safety.js](https://github.com/nodejs/node/tree/main/test/parallel/test-signal-safety.js)
-1870. [parallel/test-signal-unregister.js](https://github.com/nodejs/node/tree/main/test/parallel/test-signal-unregister.js)
-1871. [parallel/test-snapshot-api.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-api.js)
-1872. [parallel/test-snapshot-basic.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-basic.js)
-1873. [parallel/test-snapshot-cjs-main.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-cjs-main.js)
-1874. [parallel/test-snapshot-console.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-console.js)
-1875. [parallel/test-snapshot-dns-lookup-localhost-promise.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-dns-lookup-localhost-promise.js)
-1876. [parallel/test-snapshot-dns-lookup-localhost.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-dns-lookup-localhost.js)
-1877. [parallel/test-snapshot-dns-resolve-localhost-promise.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-dns-resolve-localhost-promise.js)
-1878. [parallel/test-snapshot-dns-resolve-localhost.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-dns-resolve-localhost.js)
-1879. [parallel/test-snapshot-error.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-error.js)
-1880. [parallel/test-snapshot-eval.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-eval.js)
-1881. [parallel/test-snapshot-gzip.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-gzip.js)
-1882. [parallel/test-snapshot-incompatible.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-incompatible.js)
-1883. [parallel/test-snapshot-net.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-net.js)
-1884. [parallel/test-snapshot-typescript.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-typescript.js)
-1885. [parallel/test-snapshot-umd.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-umd.js)
-1886. [parallel/test-snapshot-warning.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-warning.js)
-1887. [parallel/test-snapshot-weak-reference.js](https://github.com/nodejs/node/tree/main/test/parallel/test-snapshot-weak-reference.js)
-1888. [parallel/test-socket-address.js](https://github.com/nodejs/node/tree/main/test/parallel/test-socket-address.js)
-1889. [parallel/test-socket-options-invalid.js](https://github.com/nodejs/node/tree/main/test/parallel/test-socket-options-invalid.js)
-1890. [parallel/test-socket-write-after-fin-error.js](https:
```

Comment:
This sequence seems incorrect.

Reply:
My comment didn't show up properly, so I'll explain again. The regenerated TODO.md file seems to now contain a numbered list **and** a bulleted list. There also seems to be issues with the output related to `sequential/test-write-heapsnapshot-options.js`.